tests/framework/Optimizers/gold/Vector/opt_out.csv

```diff
@@ -0,0 +1,4830 @@
+RAVEN_sample_ID,t,x,y,ans,trajID,prefix,varsUpdate
+0,-5.0,-2.0,0.0,4802.76251971,1,0_0_0,0
```

Comment:
This file is quite huge... is there a way to shorten the test?

Reply:
Any shortened test is very difficult to observe the performance of.  If we decide we don't care what values we get, then yes I can list most of the convergence criteria.  I feel the values obtained at this rigor are important to the test.

It is also possible to only gold the final result, as with other optimizer tests, which will probably be required to pass the test on other machines.  I will look into fixing this.

Reply:
Turns out I'm wrong; this isn't a solution export, it's the target evaluation, so we can't just keep the last step.  The SolutionExport, on the other hand, is required to be a PointSet, so it doesn't show all the values of the vector variable `y`.  I don't know how to test to show that `y` was sampled as a vector other than with this file.

Reply:
Thank you for looking in this